xyz.kusma.aurora.yaml

```diff
@@ -0,0 +1,114 @@
+app-id: xyz.kusma.aurora
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: aurora.sh
+modules:
+- name: glslang
+  buildsystem: cmake
+  sources:
+  - type: git
+    url: https://github.com/KhronosGroup/glslang.git
+    tag: 7.10.2984
+- name: glfw
+  buildsystem: cmake
+  config-opts:
+  - "-DBUILD_SHARED_LIBS=1"
+  - "-DGLFW_BUILD_TESTS=0"
+  - "-DGLFW_BUILD_DOCS=0"
+  sources:
+  - type: git
+    url: https://github.com/glfw/glfw.git
+    tag: 3.3.6
+- name: glm
+  buildsystem: cmake
+  config-opts:
+  - "-DGLM_TEST_ENABLE=0"
+  - "-DCMAKE_INSTALL_LIBDIR=lib"
+  sources:
+  - type: git
+    url: https://github.com/g-truc/glm.git
+    tag: 0.9.9.3
+- name: freeimage
+  buildsystem: simple
+  sources:
+  - type: archive
+    url: http://downloads.sourceforge.net/freeimage/FreeImage3180.zip
+    sha256: f41379682f9ada94ea7b34fe86bf9ee00935a3147be41b6569c9605a53e438fd
+  build-commands:
+  - CXXFLAGS=-std=c++03 make -j
+  - make
+  - install -m644 Source/FreeImage.h /app/include
+  - install -m755 libfreeimage-3.18.0.so /app/lib
+  - ln -sf libfreeimage-3.18.0.so /app/lib/libfreeimage.so.3
+  - ln -sf libfreeimage.so.3 /app/lib/libfreeimage.so
+- name: assimp
+  buildsystem: cmake
+  config-opts:
+  - "-DASSIMP_BUILD_ASSIMP_TOOLS=0"
+  - "-DASSIMP_BUILD_TESTS=0"
+  - "-DINJECT_DEBUG_POSTFIX=0"
+  sources:
+  - type: git
+    url: https://github.com/assimp/assimp.git
+    tag: v5.2.1
+- name: bass
+  buildsystem: simple
+  sources:
+  - type: archive
+    strip-components: 0
+    url: https://web.archive.org/web/20220415095301/http://www.un4seen.com/files/bass24-linux.zip
+    sha256: 77f387762a9ee4b72da3c6581c22e50456ab7aeb7c9a7154564530a9468db655
+  build-commands:
+  - install -m 644 bass.h /app/include
+  - install -m755 x64/libbass.so /app/lib
+- name: demo
+  buildsystem: meson
+  config-opts:
+  - "--buildtype=release"
+  sources:
+  - type: git
+    url: https://github.com/kusma/even-laster-engine.git
+    branch: aurora-linux
+- name: aurora
+  buildsystem: simple
+  sources:
+  - type: file
+    path: aurora.sh
+  build-commands:
+  - install -m755 aurora.sh /app/bin
+- name: icon
+  buildsystem: simple
+  sources:
+  - type: file
+    path: xyz.kusma.aurora.svg
+  build-commands:
+  - mkdir -p /app/share/icons/hicolor/scalable/apps
+  - install -m644 xyz.kusma.aurora.svg /app/share/icons/hicolor/scalable/apps
+- name: desktop-file
+  buildsystem: simple
+  sources:
+  - type: file
+    path: xyz.kusma.aurora.desktop
+  build-commands:
+  - mkdir -p /app/share/applications/
+  - install -m644 xyz.kusma.aurora.desktop /app/share/applications/
+- name: appdata-file
+  buildsystem: simple
+  sources:
+  - type: file
+    path: xyz.kusma.aurora.appdata.xml
+  build-commands:
+  - mkdir -p /app/share/metainfo/
+  - install -m644 xyz.kusma.aurora.appdata.xml /app/share/metainfo/
+finish-args:
```

Comment:
all this section is usually *above* the modules

Reply:
Fixed